spec/unit/network/http/api/indirected_routes_spec.rb

```diff
@@ -250,6 +250,19 @@
                        %r{No supported formats are acceptable \(Accept: application/json, text/pson\)})
     end
 
+    it "raises FormatError if tries to fallback and pson serialization is not allowed" do
+      Puppet[:allow_pson_serialization] = false
+      data = Puppet::IndirectorTesting.new("my data")
+      indirection.save(data, "my data")
+      request = a_request_that_searches(Puppet::IndirectorTesting.new("my"), :accept_header => "unknown, text/pson")
+      allow(data).to receive(:to_pson).and_raise(Puppet::Network::FormatHandler::FormatError, 'Could not render to Puppet::Network::Format[pson]: source sequence is illegal/malformed utf-8')
+
+      expect {
+        handler.call(request, response)
+      }.to raise_error(Puppet::Network::FormatHandler::FormatError,
+                       %r{Failed to serialize Puppet::IndirectorTesting for 'my': Could not render_multiple to Puppet::Network::Format\[pson\]})
+    end
+
```

Comment:
Could you add a similar test for `"when finding a model instance"`? There's a `a_request_that_finds` helper